quodlibet/po/fr.po

```diff
@@ -523,9 +513,8 @@ msgid "Internet Radio"
 msgstr "Radio Internet"
 
 #: ../quodlibet/browsers/iradio.py:252
-#, fuzzy
 msgid "Downloading station list"
-msgstr "Télécharger les fichiers"
+msgstr "Télécharger la liste de station"
```

Comment:
Not sure about this one since the EN version is not in a plural form. I think you're point of view make sense even if both are fine (referring to the "quantity" or to the "quality" are both fine in French, and I guesstimate that it's the same in English here). If we'd go to changing to plural, it would first need a fix in the original EN version first since I don't like moving out the original when translating.